cy8th8-d1e97/cod-hgy7ss_cy8th8-d1e97.xml

```diff
@@ -78,19 +78,34 @@
           <lb ed="#B"/>dignitatibus lib. 12.</ref> et in <ref>L. senatoris filium. ff. de senato</ref> et de 
           <lb ed="#B"/>illa nobilitate et uirtutibus quibus ad illam predecesso<lb ed="#B"/>res
           peruenerunt posteri et descendentes gloriantur et 
-          <lb ed="#B"/>se impigriter sepe jactitant: ita si maiores sui aliquo 
+          <lb ed="#B"/>se inpigniter sepe jactitant: ita si maiores sui aliquo 
           <lb ed="#B"/>dedecore aut infamia morum presertim sunt notati: 
-          <lb ed="#B"/>uel eis aliqua injuria irrogatur: proculdubio descenden<lb ed="#B"/>tes 
+          <lb ed="#B"/>uel eis aliqua iniuria irrogatur: proculdubio descenden<lb ed="#B"/>tes 
           omnes eiusdem opprobrii confusione percelluntur: 
           <lb ed="#B"/>et eorum omnium honori detrahitur. Nam si parentibus 
           <pb ed="#B" n="6-r"/>
-          <lb ed="#B"/>jam mortuis aliquis honor exhibeatur ueluti cum laudat. 
+          <lb ed="#B"/>iam mortuis aliquis honor exhibeatur ueluti cum laudat. 
           <lb ed="#B"/>et celebratur eorum memoria manifestum est in filios et ne<lb ed="#B"/>potes 
           et pronepotes atque in omnes descendentes honorem 
           <lb ed="#B"/>eiusmodi redundare. E contrario aut si fierent eis alique 
           <lb ed="#B"/>inhonorationes puta si extumularentur aut ossa eorum cre<lb ed="#B"/>marentur
-          magna injuria uiuis et filiis et aliis descendentib. 
-          <lb ed="#B"/>et attinentibus irrogaretur.
+          magna iniuria uiuis et filiis et aliis descendentib. 
+          <lb ed="#B"/>et attinentibus irrogaretur. Similiter vero redundant in 
+          <lb ed="#B"/>parentes defunctos bonum vel malum que proveniunt filiis 
+          <lb ed="#B"/>nepotibus pronepotibus atque aliis posteris eorum scilicet et 
+          <lb ed="#B"/>prosperitates et inhonorationes infamie vel <sic>infortunia<sic/>: 
```

Comment:
@bartolomedelascasas  one issue here: take a look at your ending tag for `</sic>`. You had `<sic/>` but it should be `</sic>`

```suggestion
          <lb ed="#B"/>prosperitates et inhonorationes infamie vel <sic>infortunia</sic>: 
```